dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -0,0 +1,66 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Create/update 'Index' issues in the DTS portal from Github.",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
```

Comment:
I learned recently that the way Mike is doing this is 100% correct, and Airflow recommends that you do /not/ try to set the start date dynamically, as the start date is one of the things that it uses to identify DAGs.

Mike is establishing a great pattern here.

Reply:
I never knew why we had to include start dates in our dags on airflow1, TIL its helps they identify DAGs